uint8 channel   # 0-7
uint32 init_on_time_us    # on time for pwm duty cycle in micro-seconds
---
